modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/resources/META-INF/resources/js/edit_blueprint/ResultListItem.js

```diff
@@ -26,24 +26,25 @@ const RESULTS_DEFAULT_KEYS = [
 	'b_created',
 	'b_modified',
 	'b_author',
-	'b_assetEntryId',
 ];
-const RESULTS_HIDE_KEYS = ['explanation', 'document', 'b_viewURL'];
+const RESULTS_SHOW_KEYS = ['b_assetEntryId', 'id'];
 
 function ResultListItem({item}) {
 	const [collapse, setCollapse] = useState(true);
 
-	const _renderListRow = (property) => (
-		<ClayLayout.Row justify="start" key={`${item.id}_${property}`}>
+	const _renderListRow = (property, value, id) => (
+		<ClayLayout.Row justify="start" key={`${id}_${property}`}>
```

Comment:
```suggestion
		<ClayLayout.Row justify="start" key={property}>
```

I think property might suffice as a key here? It only needs to be unique to the immediate list it's part of.